lib/db/schema/patch-045-046.sql

```diff
@@ -0,0 +1,224 @@
+-- Add emails table and corresponding procedures to
+-- create, get and delete them.
+
+CREATE TABLE IF NOT EXISTS emails (
+  normalizedEmail VARCHAR(255) NOT NULL PRIMARY KEY,
+  email VARCHAR(255) NOT NULL,
+  uid BINARY(16) NOT NULL,
+  emailCode BINARY(16) NOT NULL,
+  isVerified BOOLEAN NOT NULL DEFAULT FALSE,
+  isPrimary BOOLEAN NOT NULL DEFAULT FALSE,
+  verifiedAt BIGINT UNSIGNED,
+  createdAt BIGINT UNSIGNED NOT NULL,
+  INDEX `uid` (`uid`)
```

Comment:
nit: please check the naming conventions used for indexes on other tables and use something consistent; it appears we favour something like `tableName_columnName_columnName` which would make this `INDEX `emails_uid` for consistency.